azavea/raster-foundry
# Make polygons visible after saving AOIs/loading AOI parameters page

## Overview

Use draw toolbar component when editing AOI on the AOI Parameters page.
Delete draw-aoi related html, module, controller, route, and state - since they are replaced by the draw toolbar component.
Make AOI polygons, if any, visible: on AOI parameters page load; after a new AOI polygon is saved; after hitting the cancel button on draw toolbar.
Add AOI polygons as one of the layers in the layer control.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

Polygons are visible on page load, and are listed as one of the layers in the control.

<img width="1680" alt="1" src="https://user-images.githubusercontent.com/16109558/27560756-689969c2-5a93-11e7-8984-8e266ecec0b5.png">

Use of the draw toolbar.
<img width="1680" alt="2" src="https://user-images.githubusercontent.com/16109558/27560758-6b915982-5a93-11e7-92e4-51a2bf052188.png">

Polygons are still there after clicking save or cancel on the toolbar.
<img width="1680" alt="3" src="https://user-images.githubusercontent.com/16109558/27560762-6d351972-5a93-11e7-98c0-bddac5093612.png">



### Notes

Thanks @Lknechtli for the help.


## Testing Instructions

 * Create a new AOI project, or go to your previous AOI project
 * Go to the AOI Parameters page
 * If you are in a previous AOI project, check if the polygons are there.
 * Click on "Edit Area of Interest"
 * Use the toolbar to create or edit an AOI
 * Click "save" or "delete" on the toolbar and check if the polygons are still there.

Closes #2031
